src/ECharts/Commands.purs

```diff
@@ -233,6 +251,9 @@ missingValue = set "" undefinedValue
 valuePair ∷ ∀ i. String → Number → DSL (value ∷ I|i)
 valuePair a b = set "value" $ toForeign [toForeign a, toForeign b]
 
+titles ∷ ∀ i. DSL TP.TitlesI → DSL (title ∷ I|i)
+titles = set "title" <<< buildArr

```

Comment:
Would it be possilbe to name the phantom type `titles` here rather than `title`? That way it keeps it distinct from values generated with the `title` combinator.

I'm not sure if that works with the way things are elsewhere, but I assume the phantom labels are named arbitrarily, and not representative of the exact names in the constructed `Foreign` type, in which case I'm pretty sure what I'm suggesting here should work, and improve the type safety of the API. 

Unless values generated by either `title` or `titles` are interchangeable in the actual echarts API, in which case ignore what I said :smile:.